src/index.ts

```diff
@@ -0,0 +1,41 @@
+export {
+  BaseAsyncIterReader,
+  AsyncIterReader,
+  LimitReader,
+  NoConcatInflator,
+  StatusAndHeadersParser,
+  StatusAndHeaders,
+  WARCParser,
+  WARCSerializer,
+  WARCRecord,
+  WARC_1_0,
+  WARC_1_1,
+  Indexer,
+  CDXIndexer,
+  postToGetUrl,
+  getSurt,
+  appendRequestQuery,
+  jsonToQueryString,
```

Comment:
Same as in lib/index.ts, need to include additional exports
```suggestion
  jsonToQueryParams,
  mfdToQueryParams,
  jsonToQueryString,
  splitChunks,
```